blueprints/jsonapi-adapter/files/__root__/__path__/__name__.js

```diff
@@ -1,17 +1,5 @@
 <%= importStatement %>
-import config from '../config/environment';
 
 export default <%= baseClass %>.extend({
   type: '<%= entity %>',
-
-  url: config.APP.API_PATH + '/<%= resource %>',
```

Comment:
We're talking about the removal of the `url` in the blueprint, right?

I guess we could keep it. It is faster performance-wise (no computed property). That's the only upside I can think of though. (still a good point)

How do you feel about the computed property as default though? I think it's very nice to have and opens up a path to completely optional service-per-resource. Since the `type` always matches the resource type, a basic service can be set up automagically in code when none is found in the app. I would very much like that. The services in my app are all dead weight that all do the exact same thing; create a url from the `type` and the `config.APP.API_` settings.